cmsdb/processes/ewk.py

```diff
@@ -312,7 +312,7 @@
     name="zz",
     id=8100,
     label="ZZ",
-    xsecs={13: Number(0.1)},  # TODO
+    xsecs={13: Number(12.13)},  # AN-19-197_v3
```

Comment:
To make lookups faster, we should perhaps add the full link here, e.g.

`https://cms.cern.ch/iCMS/jsp/db_notes/noteInfo.jsp?cmsnoteid=CMS%20AN-2019/197 (v3)`

Same below.

Reply:
Good idea, I'll add the links as you suggest.